### DISCUSSION TITLE
Failed to configure pod interface: timed out waiting for OVN port binding

### DISCUSSION THREAD

**Discussion by gabrilabs75** [okd-project/okd#1110] [General]
**Describe the bug**

The problem happened when we tried to apply new configuration that request the restart of nodes.
For example:

```
$ oc get po -n openshift-apiserver

NAME                         READY   STATUS     RESTARTS   AGE
apiserver-fhfwq   2/2     Running    2          5d2h
apiserver-twmxx   2/2     Running    0          5d2h
apiserver-vd466   0/2     Init:0/1   0          15m

```

****Version****

Client Version: 4.9.0-0.okd-2021-11-28-035710
Server Version: 4.9.0-0.okd-2022-01-14-230113
Kubernetes Version: v1.22.1-1827+e790d7fa218bce-dirty

**Log bundle**

``
28m         Warning   FailedCreatePodSandBox                       pod/apiserver-vfkpj    (combined from similar events): Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create pod network sandbox k8s_apiserver-vfkpj_openshift-apiserver
**error adding pod openshift-apiserver_apiserver-vfkpj to CNI network** "multus-cni-network": [openshift-apiserver/apiserver-vfkpj/4581fd06-fc1d-4baa-9390-825666f47f74:ovn-kubernetes]: error adding container to network "ovn-kubernetes": CNI request failed with status 400: '[openshift-apiserver/apiserver-vfkpj ] [openshift-apiserver/apiserver-vfkpj ] failed to configure pod interface: timed out waiting for OVS port binding (ovn-installed) for...
``

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Comment by Gjonni** [comment]
Hi,
I add that all the infrastructural pods that require an ip from ovnkubernetes do not start and remain in containercreating.
this problem occurs on all nodes of the cluster when we delete a pod it no longer restarts while the pods that are running and that have received an ip from ovnkubernetes remain running

this problem has been present for about 2 days and no machineconfig has been applied

**Comment by freedge** [comment]
Hi! experiencing this with OCP 4.10.3. Added a machine configuration on workers and masters, and I am hitting the same issue. How did you solve it?

EDIT: for my specific use case, there was an issue with security groups on the IaaS (OpenStack). Traffic between master nodes was dropped on some ports. (https://bugzilla.redhat.com/show_bug.cgi?id=2071272)

**Comment by gabrilabs75** [comment]
Any update about this issue?

**Comment by darren-oxford** [comment]
Try following the Rebuild all Openshift-OVN-Kubernetes pods knowledge-base article.
https://access.redhat.com/solutions/6449381

It worked for me.
